codersdk/workspaceagents.go

```diff
@@ -408,9 +419,9 @@ type WorkspaceAgentDevcontainer struct {
 	ConfigPath      string    `json:"config_path,omitempty"`
 
 	// Additional runtime fields.
-	Running   bool                     `json:"running"`
-	Dirty     bool                     `json:"dirty"`
-	Container *WorkspaceAgentContainer `json:"container,omitempty"`
+	Status    WorkspaceAgentDevcontainerStatus `json:"status"`
```

Comment:
🤔 This is a breaking API change; I would suggest deprecating the `Running` field instead.

Reply:
It should be fine, this API is not public yet and has no stability guarantees (agent v0).